src/tests/ftest/cart/util/cart_logtest.py

```diff
@@ -433,6 +433,11 @@ def _check_pid_from_log_file(self,
                     elif show and server_shutdown and line.get_msg().endswith(
                             "DER_SHUTDOWN(-2017): 'Service should shut down'"):
                         show = False
+                    elif show and server_shutdown and \
+                         line.function == 'ds_obj_ec_aggregate' and \
```

Comment:
just confirm, it only silent it when server_shutdown, and in func of "ds_obj_ec_aggregate"?

can it just silent it for all possible cases in server_shutdown? (if with node killing in the test, may also with other case not in server_shutdown get that error).

Reply:
Yes, we can remove the function check if you think that's best.

Reply:
ok, thanks.